### ISSUE TITLE
Anable to specify source.csv

### ISSUE THREAD

**Issue by vojtechhuser** [OHDSI/WhiteRabbit#80]
I would like to have an option to specify source tables not by running WhiteRabit but specify it as a CSV file.

using the save format as for the target

**Comment by aguynamedryan** [comment]
Agreed!  This issue is highly related to #45 and was discussed briefly (and seemingly ignored) [in this comment](https://github.com/OHDSI/WhiteRabbit/issues/44#issuecomment-96019521)

The issue with loading a CSV into the source tables is that you lose the summary information generated by WhiteRabbit.  Ideally, we should be able to load a WhiteRabbit report and then supplement that information with information from a source.csv file.
